proposals/simd/SIMD.md

```diff
@@ -784,3 +784,25 @@ Lane-wise saturating conversion from floating point to integer using the IEEE
 resulting lane is 0. If the rounded integer value of a lane is outside the
 range of the destination type, the result is saturated to the nearest
 representable integer value.
+
+Converts low or high half of the smaller lane vector to a larger lane vector,
+with signed or unsigned saturation.
+
+### Integer to integer narrowing
+* `i8x16.convert_s/i16x8(a: v128, b: v128) -> v128`
+* `i8x16.convert_u/i16x8(a: v128, b: v128) -> v128`
+* `i16x8.convert_s/i32x4(a: v128, b: v128) -> v128`
+* `i16x8.convert_u/i32x4(a: v128, b: v128) -> v128`
+
+Converts two input vectors into a smaller lane vector by narrowing each lane,
+signed or unsigned.
+
+### Integer to integer widening
+* `i16x8.convert_s/i8x16Low(a: v128) -> v128`
+* `i16x8.convert_s/i8x16High(a: v128) -> v128`
+* `i16x8.convert_u/i8x16Low(a: v128) -> v128`
+* `i16x8.convert_u/i8x16High(a: v128) -> v128`
+* `i32x4.convert_s/i16x8Low(a: v128) -> v128`
+* `i32x4.convert_s/i16x8High(a: v128) -> v128`
+* `i32x4.convert_u/i16x8Low(a: v128) -> v128`
+* `i32x4.convert_u/i16x8High(a: v128) -> v128`
```

Comment:
These names should be updated to be consistent with the naming scheme of the rest of the instructions. How about `i16x8.widen_{low,high}_{s,u}_i8x16` and similar? Same for the narrowing.